homeassistant/components/recorder/migration.py

```diff
@@ -206,14 +206,35 @@ def _add_columns(engine, table_name, columns_def):
 
 def _modify_columns(engine, table_name, columns_def):
     """Modify columns in a table."""
+    if engine.dialect.name == "sqlite":
+        _LOGGER.debug(
+            "Skipping to modify columns %s in table %s. "
+            "Modifying column length in SQLite is unnecessary, "
+            "it does not impose any length restrictions.",
```

Comment:
```suggestion
            "it does not impose any length restrictions",
```

Our standard is to not add a period.